provider/virt_storage/storage_volume.py

```diff
@@ -138,7 +138,10 @@ def refresh_format_by_params(self, params):
 
             backing = params.get("backing")
             if backing:
-                backing_node = "drive_%s" % backing
+                if params.get("backing_node"):
+                    backing_node = params["backing_node"]
+                else:
+                    backing_node = "drive_%s" % backing
```

Comment:
we might not need this code change, we can give the right node-name in qsd_image_format param, e.g. 
qsd_image_format = {"driver":"qcow2", "node-name": "drive_stg1"}

Reply:
Have removed this update, thanks @zhencliu